Jenkinsfile

```diff
@@ -33,14 +37,19 @@ node('master') {
 
         stage('Test') {
             wrap([$class: 'AnsiColorBuildWrapper', colorMapName: 'xterm']) {
-                sh './bin/behat -c build/behat.api.yml --colors -f pretty --strict'
-                sh './bin/behat -c build/behat.i18n.yml --colors -f pretty --strict'
+                timeout(time: 2, unit: 'HOURS') {
+                    if (env.WD_BROWSER_NAME == 'phantomjs') {
```

Comment:
Ok. I discussed with @jfhovinne and @voidtek about this comment. The case I was talking about will only happen if the server is not well configured and then, it will be treated at the environment installation.

So, that can be considerate as a nice to have that can be treated in another ticket.